setup.py

```diff
@@ -125,6 +125,7 @@ def get_version():
             "pytest >= 3.0.0, <4",
             "coverage >= 4.5.0, <5",
             'pytest-timeout >= 1.3.0, <2; platform_python_implementation!="Jython"',
+            'xonsh;python_version>"2.7"',
```

Comment:
Shouldn't this be just ``>3``? What's xonosh version support?

Reply:
Xonsh only supports Python 3 (v3.4+)

Reply:
Happy to change this if you want.

Reply:
If that's the case yes please change it so. Also we need to test that on Python 2.7 we don't generate the activation script.

Reply:
Just to be clear - you should still be able to create a Python 2.7 virtualenv *from* Python 3 in xonsh, right?

Reply:
@pganssle - yes that should still be possible.  You just can't create a vitrualenv from Python 2.7 because you wouldn't be able to run xonsh in the first place.